packages/react-jss/src/withStyles.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import React, {Component, type ComponentType, type Node} from 'react'
 import hoistNonReactStatics from 'hoist-non-react-statics'
-import {getDynamicStyles, SheetsManager, type StyleSheet} from 'jss'
+import {SheetsManager, type StyleSheet} from 'jss'
+import {getFunctionStyles} from 'jss-plugin-rule-value-function'
```

Comment:
we should not depend on the plugin directly in react-jss, because user can pass any plugin version, so you end up using a function from the wrong plugin version, thats I why I think it should bee a separate package

Reply:
We are already depending on the plugin through the preset.

Also, the values should match, as we are using the locked mode of lerna.

Reply:
preset is overridable by user, that function won't be

Reply:
Hmm, but with v10 they should always have the same version

Reply:
I was just thinking about this again. Actually  since that function is not part of the plugin module itself, it doesn't matter. So we can actually do it without introducing much confusion.

Reply:
Because we install `jss-preset-default` anyway and there is no way to tree shake that, the plugin will be imported anyway.